packages/next/src/client/components/react-dev-overlay/app/hot-reloader-client.tsx

```diff
@@ -607,12 +607,11 @@ export default function HotReload({
       const componentStackTrace =
         (error as any)._componentStack || errorDetails?.componentStack
       const warning = errorDetails?.warning
-      const stitchedError = getReactStitchedError(error)
```

Comment:
The Owner Stack at this point is the Owner Stack of this specific call which is in the Effect of `HotReload`. But that's not the owner of this error so we ended up appending two different Owner Stacks.

Reply:
Good catch, wasn't realized we shouldn't do this here. Shall we leave a comment for it?

Reply:
Not sure. I'd expect we'd need that comment everywhere we handle errors which would include all the error boundaries. I feel good leaving this as-is since we have test coverage. If we break this again, we definitely should though.